File: FinalProject/Contruction/ecobike_rental_server/TKXDPM_API/Model/Address.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TKXDPM_API.Model
{
    public class Address
    {
        [Key] public int AddressId { get; set; }
        [Column(TypeName = "varchar(255)")] public string AddressName { get; set; }
        public float Longitude { get; set; }
        public float Latitude { get; set; }
    }

    public class AddressResponse
    {
        public AddressResponse()
        {
            AddressName = "AddressName";
            Longitude = 10;
            Latitude = 10;
        }

        public string AddressName { get; set; }
        public float Longitude { get; set; }
        public float Latitude { get; set; }
    }
}

File: Programming/ecobike_rental/lib/helper/api/response/address.dart
class AddressResponse {
  String addressName;
  int longitude;
  int latitude;

  AddressResponse({this.addressName, this.longitude, this.latitude});

  AddressResponse.fromJson(Map<String, dynamic> json) {
    addressName = json['addressName'];
    longitude = json['longitude'];
    latitude = json['latitude'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['addressName'] = this.addressName;
    data['longitude'] = this.longitude;
    data['latitude'] = this.latitude;
    return data;
  }
}
